### DISCUSSION TITLE
best way to monitor message queue in milvus

### DISCUSSION THREAD

**Discussion by pycui** [milvus-io/milvus#36643] [Q&A and General discussion] [answered]
e.g. using Pulsar, is there recommendation way to monitor its usage and health in the context of milvus operation? Thanks. It's not included in attu UI. 

also, is there consideration of choosing Pulsar vs Kafka in Milvus' context? thanks

**Comment by xiaofan-luan** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/31551606-478a-4111-be1f-cd2420e0f7c4)

In the monitoring panel, you can see all the monitoring metrics for consume and create topic and many other operations. 

Both message queue works pretty well milvus so we don't have strong preference.
